domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -1059,6 +1083,12 @@ class ProfileManagementController @Inject constructor(
             "Profile cannot be an admin"
           )
         )
+      ProfileActionStatus.PROFILE_TYPE_UNKNOWN ->
+        AsyncResult.Failure(
+          UnknownProfileTypeException(
+            "ProfileType must be set"
+          )
+        )
```

Comment:
```suggestion
        AsyncResult.Failure(UnknownProfileTypeException("ProfileType must be set."))
```

Small reformatting suggestions.

Reply:
Fixed